From ae0ac04ac2a301db002623ac44c342ea62c9d066 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Wed, 26 May 2021 20:47:24 +0300
Subject: [PATCH] [SW-34387] gaudi2: add dummy keys

since common its files requires key, this patch adds keys
to gaudi2 folder.

Change-Id: I5308b49be380ba0cd673708751a7a0a69c09b37c
---

diff --git a/board/habanalabs/gaudi2/keys/hl_rsa3072.dtb b/board/habanalabs/gaudi2/keys/hl_rsa3072.dtb
new file mode 100644
index 0000000..9bb736b
--- /dev/null
+++ b/board/habanalabs/gaudi2/keys/hl_rsa3072.dtb
Binary files differ
diff --git a/board/habanalabs/gaudi2/keys/hl_rsa3072.key b/board/habanalabs/gaudi2/keys/hl_rsa3072.key
new file mode 100644
index 0000000..0b0b5e8
--- /dev/null
+++ b/board/habanalabs/gaudi2/keys/hl_rsa3072.key
@@ -0,0 +1,40 @@
+-----BEGIN PRIVATE KEY-----
+MIIG/AIBADANBgkqhkiG9w0BAQEFAASCBuYwggbiAgEAAoIBgQDF2V301XtknprL
+IQTNqX2sokhEvCZipwjkrz8CpY058eR+8AFv2n9yd3KdfYclSBFksbJUH96ULD/b
+mKcg318OeQWhLsxL01SVY/yAxSjHGdy4uoqpGiVMYdeX/z/6qoPnvBn7kNGFiHlz
+SJxU4bOl3kvbC9Gc3wUk+41IwbUSAN6Xcf2sqVnDPyxgGTuPPpc7OEAd4mulKh4K
+PBPlt4MUVaDfaFRSbxQh0tAq8E0SSbW8UEGIaa7/vMo7FxOzfgaAtWXZfAIBSJvB
+BCBcM3eAp7lxs5xgw+BJYS3cj/UBU90oftaZhisYTnHIekTYnX9t1/n2n84MIw5l
+/LWTHBavsqZyEAlX3mkakXwvtXZFXsVoaBhKa6YC+8315E9Q9Y4NTdX+C5zCoApq
+aFXdq0093/flxh/4rca2PWiBGQzy3rzjqh/BQX35bH5s/mJeSqY3SqBcofve0Jb2
+iz/3OR8Is0GKR57Q25BUHAQ7cLLfXfawraApz0N2rQLo09PjfhcCAwEAAQKCAYA4
+CouKE8RJaoGXIwmsxaMi+K62xYKr8/tZZ2hWNBWOQb3C1PHbQI1UG5B8GrfisAi5
+OHo2SPavit8lsRY6Aft3aoFVOB/HJJuzWyiFQxN7XhB+xNwM7Sp+ecJrO/Jf6tfF
+FqtXbhxjEhawuHMLugi8ZXygFbdwk11uLdDFuJS0/kkOpcvAkhGqc+T6bXtI5mZI
+8HR84osyM/1hUcYjDfmTk3OnhHJKDUurDv9HAl0aXoWbSzJc1O6sYsn3teAvOfVW
+Lt+pNSZ88JZysUCIgREu0CY6aGzaptgSfrxCw2Suequ/I3yGLhzfslZ1dx9dKmj2
+/8TX4z+uosjtAO4Z45FePrDbdiNPG2NaWLxAtaxYaXIHDU/LlIZSsZ1G9rfKkM59
+I1TqhDzH8/stFZOx4o1TXIJkB1K8YNzvBE9J6+hRw9YJ0sLy8OTDz8/qE1a4TxWb
+zjcZeSAygOzVcnAHT8K5NVFH8OhMYDTr416Kt4kl7am0qZdui9q+Aczp86tMUaUC
+gcEA/XbqSTZSfUYgDcK+GXPY9CTbgSlKcUzkebNlyWrJt6JrTP7ImNJLQhcEwE5C
+hLMGCpq25D9Dx3e2axnPgtkxHgBl16ekHsWDXvPsJQAd2R+A0YOWG6908EiPSfXM
+6mzcCPgRplfs0kmxhXUDH6TKOyhJy7lB63goR+k34+Tiz/CpvPX2TiOrtDEWinmh
+kCffM0qPryKsYW/U/RWHZRysCHZosh76D3oMRPw0ZoqgWxPJZz1ZVKSniyttmiVP
+oesjAoHBAMfUB27ABn26IgRHW+EMXifgl/HzpDObmeooxMPciQfTKM/PPpISxeD4
+Xd86Dpx1AwLf/DbKJ1m93QwpMb+QWPo/iN7Ecz6eHTyFX2yWZhTG4K8lOr4Kp3Kd
+Li8ZoPaYWMsLe9IHR/u+zTdhNYADVkhFrWspvbXIf0lqTHtDO/qeQhI4gW/rVN0Z
+Uce74i0bWjQDqnBxkqQNPqKt6ujXrrGMaxwaGubQrxPGSOzDGfpYkMDW7Myq5CgR
+DAM0g/d6fQKBwCFcb4XnYoAepMVq5n6PIvUvjLy5ZJT5G/FwzWrvZVGITTs7JcgH
+leRDB1sQcezpa1gl+QCd2xv8A8ShBxEf6TkbsEvsQsvSD3KGBITyOPuFC/2HplBK
+jce6pTGE6s3IOkXr8NbGJ+Od04HPXvPhSsTyjpQD9Es764FbjwGt9LSN5ql6bkrm
+owj6lQ7w53UD6nrjy3mvxKSmhZAyLzn9xh7vBuS68wrqc4b48yS0Bls2hYHTL1jW
+uz60Hp71Xj/fpQKBwEJr3+yYN5AjBpuVfJzZl5J5vQfXjIT6+7YxxvWzoHjRX4dz
+w8CaLNwSsmE40xJ+cnnKKfOsTlguHAOmzbp3KJxF8iDZ+9ScietMCeP2Ogg98wFE
+xrnUoMQgKvHM+wv8p8cWEhpGAU9UpLH7xeS/Nzbesj4kqcJ62pR0mOd/+bfcUunr
+jlq9RoAXom9ZrFyB8wHHFig6GBwfquo4FRVAW/j4S8ITi02doBRanHLWhxQHzFXN
+GvkM9Lm8sNl/4z2noQKBwBsiHbut7Z53fFxVk7/J4gbq0JooXg4QyoKZCQfLkT6w
+1bvs8TSAIE8tplOnzxe10fZwBNUR3gJj88Os2pwfpC52hQn3KeQAhL/9YSqNtcTN
+xTYo3Z7xzyv0NM0Sqii/1MQIPW42j4yTOjsbllUS4jj4jHV8YAgVAeKm5PebtC+2
+uAZc/BQJZaZG/wTs22f886wqOC86funWPXDuLJ3SASn9V+kIee+m4ZDr8rdHvNxQ
+7KHpssorVYrLiSCTKiU2ug==
+-----END PRIVATE KEY-----
